cvm-attestation/AttestationClient.py

```diff
@@ -163,13 +164,22 @@ def attest_guest(self):
         hardware_evidence = self.get_hardware_evidence()
         hw_report = hardware_evidence.hardware_report
         runtime_data = hardware_evidence.runtime_data
+        report_type = hardware_evidence.type
 
-
-        # Logs important SNP fields from the hardware report
-        self.log_snp_report(hw_report)
```

Comment:
do you still print the snp report fields?

Reply:
Yes, this is moved to line 140 above